### ISSUE TITLE
Adjust CSS styling of rich text toolbar to Bootstrap 4 for consistency

### ISSUE THREAD

**Issue by jywarren** [publiclab/PublicLab.Editor#664]
I think the lines of code could be somewhere around here? 

https://github.com/publiclab/PublicLab.Editor/blob/cf1fc4a9c4b70afaede856bac214e27388ab6cfa/src/adapters/PublicLab.Woofmark.js#L372

![image](https://user-images.githubusercontent.com/24359/100797009-78b10700-33ef-11eb-8333-12c2aa5a8d7b.png)

See how the buttons here don't match -- i think the basic buttons are via `woofmark` but the black-outlined ones are the ones we've added programmatically after `woofmark` is loaded. We'd love help with this, it probably will take some trial and error but isn't such a complex change!

_Originally posted by @jywarren in https://github.com/publiclab/PublicLab.Editor/issues/663#issuecomment-736821577_ (cc @keshav234156, noting I've split this out!)

**Comment by eynglv** [comment]
@jywarren hi, could i take this bug? would be my first open source contribution

are you looking to change the outline (make all the buttons outlined) or you want them to be basic (via woofmark)?

**Comment by devan5hu** [comment]
@jywarren Hello! I am fairly new to contributing to major repositories on Github in-fact this is my very first comment. I started working on this issue today and found out the fix for it. Just wanted to share a picture of the result before I created a new branch.   
![Buttonfix](https://user-images.githubusercontent.com/52581680/107544161-f3688080-6bef-11eb-9485-d0a3cc260379.png)

I also fixed the `Custom Insert` button in responsive view. 
![Buttonfixresponsive](https://user-images.githubusercontent.com/52581680/107544447-3fb3c080-6bf0-11eb-99d3-c64d228f50ae.png)

Please correct me if I am wrong and you expected something else!

**Comment by RaviAnand111** [comment]
hey @jywarren @TildaDares @NARUDESIGNS, can I work on this issue ?

**Comment by TildaDares** [comment]
@RaviAnand111 I think this issue has been resolved. I checked on my local environment and all of the buttons match.

**Comment by TildaDares** [comment]
Thanks @RaviAnand111! I think we can close this issue now.

**Comment by RaviAnand111** [comment]
Hey @TildaDares, I checked my local environment and all the buttons do not match,
![image](https://user-images.githubusercontent.com/59113935/147956242-43498b3f-25da-4a82-b3b7-60d882591a25.png)
Do you know why is this happening?

**Comment by TildaDares** [comment]
@RaviAnand111 Have you pulled changes from this repo recently?

**Comment by RaviAnand111** [comment]
yes I did, git status, says uptodate see 
![image](https://user-images.githubusercontent.com/59113935/147957034-f4e4ee56-a99d-481c-afd3-5752c8239e1f.png)

**Comment by RaviAnand111** [comment]
see this the result of git pull upstream main
![image](https://user-images.githubusercontent.com/59113935/147957664-d4516f01-5ea0-4f64-a0f3-524a930b6654.png)

**Comment by TildaDares** [comment]
@RaviAnand111 Did you do a `git fetch upstream` then a `git merge upstream/main`?

**Comment by TildaDares** [comment]
@RaviAnand111 I can see what you see now. I just updated my local repo.

**Comment by RaviAnand111** [comment]
No, I didn't, Should I do it right now ?

**Comment by TildaDares** [comment]
I've just assigned you @RaviAnand111

**Comment by RaviAnand111** [comment]
Thanks @TildaDares , 
I just had this 1 doubt, should I create a different branch every time I work on different issue or should I work in the main branch ?

**Comment by TildaDares** [comment]
@RaviAnand111 It's better to create a different branch when you work on different issues so that you can isolate your changes from your main branch.

**Comment by RaviAnand111** [comment]
ok, now I understand, ThankYou for explaining, you are like an elder sister who helps me whenever I am in trouble, 
ThankYou

**Comment by NARUDESIGNS** [comment]
> ok, now I understand, ThankYou for explaining, you are like an elder sister who helps me whenever I am in trouble, ThankYou

Lol yeah, she's amazing. In my case, she'd always help me with very useful resources.
You are doing a great job as well my friend @RaviAnand111

**Comment by RaviAnand111** [comment]
Thanks alot, @NARUDESIGNS, it is really a complement, getting thumps up from you guys. ❤️

**Comment by RaviAnand111** [comment]
Does this look good ? @TildaDares @NARUDESIGNS 

![image](https://user-images.githubusercontent.com/59113935/148033878-7592beb3-587a-4647-b643-3c9efdb195c3.png)

**Comment by RaviAnand111** [comment]
Closer look 

![image](https://user-images.githubusercontent.com/59113935/148034930-2b414b1e-0c2e-43a1-bda4-75b6dd8bc026.png)

**Comment by NARUDESIGNS** [comment]
@RaviAnand111 We've not decided whether or not to go with the borders. I'm working on creating some mockups which the team has to review and dicuss the direction we should take from their. If you are interested in joining me, that's fine. We can communicate on gitter and share ideas about the UI of the editor.
This is a very good job by the way but we need to have a revision and discussion before a decision. 
Well done Ravi!

**Comment by RaviAnand111** [comment]
yeah, #705 has already solved it, so should I leave it and work on some other issue ?, 
yeah, sure you can tell me about UI changes if you want. 
Thanks
